src/mach/load_command.rs

```diff
@@ -1014,18 +1056,18 @@ pub struct VersionMinCommand {
 }
 
 impl VersionMinCommand {
-    pub fn new(is_ios: bool) -> Self {
+    pub fn new(platform: Platform) -> Self {
         VersionMinCommand {
-            cmd: if is_ios {
-                LC_VERSION_MIN_IPHONEOS
-            } else {
-                LC_VERSION_MIN_MACOSX
-            },
+            cmd: platform as u32,
```

Comment:
Is this a new feature ? How is this cast working since Platform isn’t repr(u32).

Reply:
Yep, this is part of the API breakage -- `VersionMinCommand` can be produced from one of four different `LC_*` constants, which are represented by that enum (since they might be useful elsewhere).

As for why it's working: I *thought* it was because I had implemented `From<Platform> for u32`, but I tried removing that trait `impl` and it still compiled :worried:. So I'll change it to `platform.into()` to make sure it's using the trait explicitly.